src/locale/messages.ts

```diff
@@ -0,0 +1,19 @@
+import en from "./en";
+import vi from "./vi";
+
+const getMessages = (locale: string) => {
+  switch (locale) {
+    case "en":
```

Comment:
Should use constant LANG_KEY from src/constants/locale.ts instead of using the raw string